bin/make-zip.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# Build a installable plugin zip
+
+output 2 "Creating archive... 🎁"
```

Comment:
```
./bin/make-zip.sh: line 5: output: command not found
```
Make sure to include this function in the script.

```
# Output colorized strings
#
# Color codes:
# 0 - black
# 1 - red
# 2 - green
# 3 - yellow
# 4 - blue
# 5 - magenta
# 6 - cian
# 7 - white
output() {
  echo "$(tput setaf "$1")$2$(tput sgr0)"
}
```

Reply:
@psealock Nice catch. Added in `74b5113`